package.json

```diff
@@ -55,7 +52,6 @@
   "devDependencies": {
     "ava": "^3.15.0",
     "execa": "^5.1.1",
-    "husky": "^7.0.4",
```

Comment:
It's not something I use, but this makes sure precommit runs before commit, even though it is not explicitly referenced.

Reply:
Never heard of something like this. Perhaps this should be documented? The readme could have a sentence or two on Husky support.

Reply:
Reinstalled husky. Dropped minimist as it didn't seem to serve any function. All test pass with or without it.

Reply:
Husky is only for developers. I'd be fine if you remove this. I added xo to CI because so many people weren't running it. :)